lib/common/utils.js

```diff
@@ -254,7 +254,7 @@ function cleanChildrenRoutes(routes, isChild = false) {
   return routes
 }
 
-exports.createRoutes = function createRoutes(files, srcDir, pagesDir) {
+exports.createRoutes = function createRoutes(files, srcDir, pagesDir, strictRoutes, caseSensitive) {
```

Comment:
Maybe extract them into a options parameter is better for future extensible.

Reply:
Done, I group all options on https://github.com/nuxt/nuxt.js/pull/3448/commits/cfe2c73cb4efe616a20a3afeb9f0cba83d95a976#diff-b0dc205b0a1997433272d8c3ea0f5e86R312